app/mailers/foreman_openscap/policy_mailer.rb

```diff
@@ -0,0 +1,41 @@
+module ForemanOpenscap
+  class PolicyMailer  < ::ApplicationMailer
+
+    def policy_summary(options = {})
+      set_url
+      user = ::User.find(options[:user])
+      @time = options[:time] || 1.day.ago
+
+      @policies = Scaptimony::Policy.all.reject { |policy| policy.assets.empty? }
+      @compliant_hosts = @policies.map { |policy| policy.assets.comply_with(policy) }.flatten.map(&:host)
+      @incompliant_hosts = @policies.map { |policy| policy.assets.incomply_with(policy) }.flatten.map(&:host)
+      changed_hosts_of_policies(@policies)
+
+      if user.nil? || user.mail.nil?
+        logger.warn "User with valid email not supplied, mail report will not be sent"
+      else
+        set_locale_for(user) do
+          subject = _("Scap policies summary")
+          mail(:to => user.mail, :subject => subject)
+        end
+      end
+    end
+
+    private
+
+    def changed_hosts_of_policies(policies)
+      hash = @policies.inject({}) do |result, policy|
+        result[policy.id] = policy.hosts
```

Comment:
Please note that this will fail if the host is deleted. See this [workaround](https://github.com/theforeman/foreman_openscap/blob/f4a4d624a76e10838abbc3714d2cdd014b0276be/app/helpers/concerns/foreman_openscap/hosts_helper_extensions.rb#L46) to solve this